src/components/Integrations/DopeBox.tsx

```diff
@@ -67,72 +72,75 @@ const DopeBox = () => {
                     you must configure incoming webhooks in your third-party
                     applications.
                   </Text>
-                  (Communications tab)
-                  <TextList className="pf-u-ml-0">
-                    <TextListItem>
-                      <Link
-                        to="https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/integrating_the_red_hat_hybrid_cloud_console_with_third-party_applications/assembly-configuring-insights-integration-with-slack_integrations"
-                        target="_blank"
-                      >
-                        Configure Slack
-                      </Link>
-                    </TextListItem>
-                    <TextListItem>
-                      <Link
-                        to="https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/integrating_the_red_hat_hybrid_cloud_console_with_third-party_applications/assembly-configuring-integration-with-gchat_integrations"
-                        target="_blank"
-                      >
-                        Configure Google Chat
-                      </Link>
-                    </TextListItem>
-                    <TextListItem>
-                      <Link
-                        to="https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/integrating_the_red_hat_hybrid_cloud_console_with_third-party_applications/assembly-configuring-integration-with-teams_integrations"
-                        target="_blank"
-                      >
-                        Configure Microsoft Teams
-                      </Link>
-                    </TextListItem>
-                  </TextList>
-                  (Reporting tab)
-                  <TextList className="pf-u-ml-0">
-                    <TextListItem>
+                  {category === IntegrationCategory.COMMUNICATIONS && (
```

Comment:
we could clean this up a bit

idea 1) put links and their labels to an array of objects and map it instead of manually copying the <TextListItem>
idea 2 - more optional) create a mapper that would return appropriate "Configure applications" links according to a category passed